extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -149,10 +150,14 @@ export class CookieWriter {
       return BASE_CID_MAX_AGE_MILLIS;
     }
 
-    // convert cookieMaxAge (sec) to milliseconds
-    return (
-      Number(inputConfig['cookieMaxAge']) * 1000 || BASE_CID_MAX_AGE_MILLIS
-    );
+    const cookieMaxAgeNumber = Number(inputConfig['cookieMaxAge']);
+    if (!cookieMaxAgeNumber) {
+      user().error(TAG, 'invalid cookieMaxAge, falling back to default value');
```

Comment:
nit: print out the invalid value as well and default value as well. 
`invalid cookieMaxAge %s, fallback to default value (1 year), inputConfig['cookieMaxAge']`